.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
Please add this to your local global git ignore. It's unnecessary to have it here.

You can read more about it here: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html